composer.json

```diff
@@ -40,6 +40,7 @@
         "doctrine/event-manager": "^1.2|^2",
         "doctrine/persistence": "^2|^3",
         "twig/twig": "^2.13|^3.0.4",
+        "pear/crypt_gpg": "^1.6",
```

Comment:
Adding that as a required dependency is a no-go to me. This must stay an optional feature.

Reply:
Okay, thought so too. But without it the unit test breaks. Is there another solution to make them work?

Reply:
that's what dev requirements are for.